### PR TITLE
feat(komodo-ecosystem): add tickers for homepage

### PR DESCRIPTION
**Pull request by Milerius**

### CODE REVIEW

File: ecosysboard/http/coinpaprika.go [hunk 1]
```diff
@@ -17,6 +17,8 @@
 package http
 
 import (
+	"encoding/json"
+	"github.com/kpango/glg"
```
Comment (ca333) [inline ecosysboard/http/coinpaprika.go]: we should ensure that external & 3rd party packages are safe/secure to use or just add a disclaimer/warning if any are un-reviewed.

File: ecosysboard/http/komodo.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+import (
+	"encoding/json"
+	"github.com/kpango/glg"
+	"github.com/valyala/fasthttp"
+	"net/http"
+	"strings"
+)
+
+var komodoCoinsToCoinpaprikaRegistry = map[string]string{
```
Comment (ca333) [inline ecosysboard/http/komodo.go]: it probably makes sense to import the coins from a `esb_config.json` file please

**Review by ca333 — APPROVED** [review]

File: ecosysboard/http/komodo.go (hunk 2 above)
Comment (Milerius) [reply ecosysboard/http/komodo.go]: yes i will do it

File: ecosysboard/http/coinpaprika.go (hunk 1 above)
Comment (Milerius) [reply ecosysboard/http/coinpaprika.go]: Yes we should proceed a full review for that

**Pull request merged**